app.js

```diff
@@ -2,6 +2,28 @@ const express = require("express");
 const app = express();
 const port = 3000;
 
+
+const MongoClient = require('mongodb').MongoClient
+
+const url = 'mongodb://127.0.0.1:27017'
```

Comment:
add a ` // TODO:` above this to move into an enmv file and get the proper dev or prod url depending on environment